src/styles/executive.css

```diff
@@ -0,0 +1,29 @@
+body
+{
+    background-color: black;
+    color: white;
+    text-align: center;
+}
+
+li
+{
+    list-style: none;
+}
+
+ul
+{
+    padding: 0px;
+}
+
+div
+{
+    display: flexbox !important;
```

Comment:
This is a broad-spanning css rule. So as to not affect every div on the site, can you be more specific here? Use div.[classname] or [containing-element] div, please and thank you.

Reply:
The display: flexbox wasn't working, this was just to test that, I removed it.